IFC4x3/Properties/l/LensDiffuserOrientation_17lLkbAkr2TPS5BNTPqqUI/Documentation.md

```diff
@@ -0,0 +1 @@
+Orientation the lens diffuser has to have.
```

Comment:
Could this be expanded in the same manner that `HotStripOrientation`?

Reply:
Updated: https://github.com/bSI-InfraRoom/IFC-Specification/pull/179/commits/3471cf4fba3a9585bdeb8395780ec9a674ace749